extensions/arc/src/ui/dashboards/postgres/postgresParametersPage.ts

```diff
@@ -0,0 +1,518 @@
+		}).component();
+
+		this.disposables.push(
+			this.saveButton.onDidClick(async () => {
+				try {
+					await vscode.window.withProgress(
+						{
+							location: vscode.ProgressLocation.Notification,
+							title: loc.updatingInstance(this._postgresModel.info.name),
+							cancellable: false
+						},
+						async (_progress, _token): Promise<void> => {
+							//Edit multiple
+							// azdata arc postgres server edit -n <server group name> -e '<parameter name>=<parameter value>, <parameter name>=<parameter value>,...'
+							try {
+								this.engineSettingUpdates!.forEach((value: string) => {
+									this.engineSettings += value + ', ';
+								});
+								await this._azdataApi.azdata.arc.postgres.server.edit(
+									this._postgresModel.info.name, { engineSettings: this.engineSettings + `'` });
+							} catch (err) {
+								// If an error occurs while editing the instance then re-enable the save button since
+								// the edit wasn't successfully applied
+								this.saveButton!.enabled = true;
+								throw err;
+							}
+							await this._postgresModel.refresh();
+						}
+					);
+
+					vscode.window.showInformationMessage(loc.instanceUpdated(this._postgresModel.info.name));
+
+					this.engineSettings = `'`;
+					this.engineSettingUpdates!.clear();
+					this.discardButton!.enabled = false;
+
+				} catch (error) {
+					vscode.window.showErrorMessage(loc.instanceUpdateFailed(this._postgresModel.info.name, error));
+				}
+			}));
+
+		// Discard
+		this.discardButton = this.modelView.modelBuilder.button().withProperties<azdata.ButtonProperties>({
+			label: loc.discardText,
+			iconPath: IconPathHelper.discard,
+			enabled: false
+		}).component();
+
+		this.disposables.push(
+			this.discardButton.onDidClick(async () => {
+				this.discardButton!.enabled = false;
+				try {
+					// TODO
+					// this.parametersTable.data = [];
+					this.engineSettingUpdates!.clear();
+				} catch (error) {
+					vscode.window.showErrorMessage(loc.pageDiscardFailed(error));
+				} finally {
+					this.saveButton!.enabled = false;
+				}
+			}));
+
+		// Reset
+		this.resetButton = this.modelView.modelBuilder.button().withProperties<azdata.ButtonProperties>({
+			label: loc.resetAllToDefault,
+			iconPath: IconPathHelper.reset,
+			enabled: true
+		}).component();
+
+		this.disposables.push(
+			this.resetButton.onDidClick(async () => {
+				this.resetButton!.enabled = false;
+				this.discardButton!.enabled = false;
+				try {
+					await vscode.window.withProgress(
+						{
+							location: vscode.ProgressLocation.Notification,
+							title: loc.updatingInstance(this._postgresModel.info.name),
+							cancellable: false
+						},
+						async (_progress, _token): Promise<void> => {
+							//all
+							// azdata arc postgres server edit -n <server group name> -e '' -re
+							try {
+								await this._azdataApi.azdata.arc.postgres.server.edit(
+									this._postgresModel.info.name, { engineSettings: `'' -re` });
```

Comment:
what's the -re from? Should that be a parameter on the edit call?

Reply:
To reset all the parameters the command is: azdata arc postgres server edit -n <server group name> -e '' -re

Reply:
That seems like something that should be a parameter on the API call then

Reply:
Will fix on next PR working on parameters page functionality.